### PR TITLE
Fixing issue #563

### PR DESCRIPTION
**Pull request by Joeu**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_
Fix for issue #563 

### This PR should:
- [x] Add skip button to both Offer Help and Need Help flows, and the skip button should redirect to /feed.
- [x] The size of the submit button should be the same as the container of the enter your email address.
- [x] Submit button should initially be disabled and only become available once a correct email has been put in the field
- [x] All the NEXT buttons should become white when hovered over the same as the Submit button for both Need Help and Offer Help

### CODE REVIEW

**Comment by robinv85** [comment]
yeah i dunno code looks ok what's the problem @idjevm ?

File: client/src/components/StepWizard/WizardSubmit.js [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
-border-radius: 4rem;
-margin-right: 7rem;
-margin-left: 7rem;
+  border-radius: 4rem;
+  align-self: center;
   cursor: pointer;
   color: ${white};
   padding: 0 0.1rem;
   background-color: ${royalBlue};
+  width: inherit;
```
Comment (robinv85) [inline client/src/components/StepWizard/WizardSubmit.js:13]: is this the same as 100%? or maybe can just remove, not sure, i never use this

**Comment by robinv85** [comment]
@idjevm the last isn't needed I think

>  All the NEXT buttons should become white when hovered over the same as the Submit button for both Need Help and Offer Help

because we discussed this before on slack with designers, and this was not really part of the design
I think we want to change it to a lighter color on hover / click

**Comment by idjevm** [comment]
> @idjevm the last isn't needed I think
> 
> > All the NEXT buttons should become white when hovered over the same as the Submit button for both Need Help and Offer Help
> 
> because we discussed this before on slack with designers, and this was not really part of the design
> I think we want to change it to a lighter color on hover / click

So @ppagarawal perhaps I can get some input here?

**Comment by ppagarawal** [comment]
> > @idjevm the last isn't needed I think
> > > All the NEXT buttons should become white when hovered over the same as the Submit button for both Need Help and Offer Help
> > 
> > 
> > because we discussed this before on slack with designers, and this was not really part of the design
> > I think we want to change it to a lighter color on hover / click
> 
> So @ppagarawal perhaps I can get some input here?

I think that to maintain that consistency we can have the next button change it to a lighter color on hover / click interaction as @robinv85 mentioned. Currently, mouse over hides the text on the button.

image

**Pull request merged**